test/runFIT.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash -ex
+
+############################################
+# Global variable
+############################################
+RACKHD_TEST_DIR="$( cd -P "$( dirname "$0" )" && pwd )"
+
+#############################################
+#
+# Usage
+############################################
+Usage(){
+    set +x
+    echo "Function: This script is used to set up environment for FIT and run FIT."
+    echo "Usage: $0 [OPTIONS]"
+    echo "  OPTIONS:"
+    echo "    Mandatory options:"
+    echo "      -w, --WORKSPACE: The directory of workspace( where the code will be cloned to and staging folder), it's required"
+    echo "      -p, --PASSWORD: password of current user which is used for FIT to log into host by, it's required."
+    echo "    Optional options:"
+    echo "      -g, --TEST_GROUP: test group of FIT, such as imageservice, smoke"
+    set -x
+}
+
+
+#############################################
+#
+#  Create the virtual env for FIT  
+#
+############################################
+setupVirtualEnv(){
+    pushd ${RACKHD_TEST_DIR}
+    virtual_env_name=FIT
+    rm -rf .venv/$virtual_env_name
+    ./mkenv.sh $virtual_env_name
+    source myenv_$virtual_env_name
+    popd
+}
+
+####################################
+#
+# 1. Modify FIT config files , to  using actual DHCP Host IP instead of 172.31.128.1
+#
+##################################
+setupTestsConfig(){
+    echo "SetupTestsConfig ...replace the 172.31.128.1 IP in test configs with actual DHCP port IP"
```

Comment:
Talked with @PengTian0, it's not elegant to replace every 172.31.128.1  . And it's possible that the RackHD interface to FIT is not 172.31.128.xx (if running in diff docker/vm)
and  there's a story RAC-5418 to track the enhancement for FIT, to improve the hard code rackhd IP.